release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1208,7 +1208,14 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
 
 You can now use the Cloud Credential Operator (CCO) utility `ccoctl` to configure the CCO to use the Google Cloud Platform Workload Identity. When the CCO is configured to use GCP Workload Identity, the in-cluster components can impersonate IAM service accounts using short-term, limited-privilege security credentials to components.
 
-//For more information, see xr/ef:../authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc#cco-mode-gcp-workload-identity[Using manual mode with GCP Workload Identity].
+For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc#cco-mode-gcp-workload-identity[Using manual mode with GCP Workload Identity].
+
+[IMPORTANT]
+====
+In {product-title} 4.10.8, support for using GCP Workload Identity for the image registry is removed from {product-title} 4.10, and you must configure the image registry to use long-lived credentials. This mitigation is planned to last until Workload Identity support for the image registry is restored in a later release.
```

Comment:
I might suggest rewording this to and split this into 2 sentences (as it was a bit confusing and it would be nice if we could give a reason why):

In {product-title} 4.10.8, image registry support is removed when you use GCP Workload Identity [due to ???]. To use the image registry in {product-title} 4.10.8, you must configure the image registry to use long-lived credentials instead. This mitigation is planned to last until Workload Identity support for the image registry is restored in a later release.